src/docs/product/crons/getting-started/cli.mdx

```diff
@@ -59,6 +59,29 @@ sentry-cli monitors run my-monitor-slug -- python path/to/file.py
 sentry-cli monitors run my-monitor-slug -- node path/to/file.js
 ```
 
+### Configuring Your Monitor via the CLI
+
+You can also use the Sentry CLI to configure your cron monitor when you run your job. This way, you can avoid having to first set up the monitor through the Sentry web interface.
+
+Configure the cron monitor by providing the cron schdule in crontab format using the `--schedule` or the equivalent `-s` argument when executing the `sentry cli monitors run` command. Please make sure to enclose the schedule in quotes, so that your shell parses the argument correctly, like so:
+```bash
+sentry-cli monitors run --schedule "<expected schedule>" <monitor-slug> -- <command> <args>
+```
+
+When providing the `--schedule` argument, we also support the following optional arguments to provide additional configuration:
+  - `--check-in-margin`: The allowed margin of minutes after the expected check-in time that the monitor will not be considered missed for.
+  - `--max-runtime`: The allowed duration in minutes that the monitor may be in progress for before being considered failed due to timeout.
+  - `--timezone` *(beta, use cautiously!)*: A tz database string (e.g. "Europe/Vienna") representing the monitor's execution schedule's timezone. Please be **absolutely sure that you provide a valid tz database string** when using this argument; as of writing, the CLI lacks the capability to validate this string, and so **the monitor creation will silently fail when the string is invalid!** For now, we recommend verifying that the cron monitor was created successfully in Sentry when using this argument.
```

Comment:
Since this `--timezone` option can cause the entire monitor upsert to silently fail if an invalid value is provided, I have labelled it as a beta feature for now. Is this appropriate?

We would need to make sure we remove the "beta" notice and the warning about the CLI not validating the string once we close getsentry/relay#2718.